docs/general/administration/hardware-acceleration/amd.md

```diff
@@ -179,7 +179,7 @@ Please refer to [this section](/docs/general/administration/hardware-acceleratio
 
 A 64-bit Linux distribution is required. **The supported GPU varies by kernel and firmware versions.**
 
-### Known Issues And Limitations On Linux
+### Known Issues And Limitations Of Linux
```

Comment:
I'd say that `On` is correct

Reply:
Thanks for the review, made the necessary change !